packages/@react-aria/combobox/stories/example.tsx

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {DismissButton, useOverlay} from '@react-aria/overlays';
+import {FocusScope} from '@react-aria/focus';
+import React from 'react';
+import {useButton} from '@react-aria/button';
+import {useComboBox} from '@react-aria/combobox';
+import {useComboBoxState} from '@react-stately/combobox';
+import {useFilter} from '@react-aria/i18n';
+import {useListBox, useOption} from '@react-aria/listbox';
+
+export function ComboBox(props) {
```

Comment:
You can essentially ignore this code, copy pasta'd from our aria hook example. If we feel the docs example is sufficient for testing I can remove this, but feels useful for when we wanna test some more hook specific behavior in the future (e.g. non virtualized combobox)

Reply:
I like this. I think we should probably have more stories for the docs example. Also helps us ensure TS actually passes 😉